deps/common/src/logseq/common/path.cljs

```diff
@@ -0,0 +1,292 @@
+(ns logseq.common.path
+  "Path manipulation functions, use '/' on all platforms.
+   Also handles URL paths."
+  (:require [clojure.string :as string]
+            [logseq.graph-parser.util :as gp-util]))
```

Comment:
`logseq.graph-parser` is required here, and do we need to add it to deps/common/deps.edn?